websites/W/watchmovieshd/presence.ts

```diff
@@ -0,0 +1,100 @@
+const presence = new Presence({
+		clientId: "1013927311499874304",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let iFrameData: {
+	currTime: number;
+	duration: number;
+	paused: boolean;
+} = null;
+
+presence.on(
+	"iFrameData",
+	(data: { currTime: number; duration: number; paused: boolean }) => {
+		iFrameData = data;
+	}
+);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			startTimestamp: browsingTimestamp,
+			largeImageKey: "logo",
+		},
+		{ pathname, href } = document.location,
+		[buttons, image] = await Promise.all([
+			presence.getSetting<boolean>("buttons"),
+			presence.getSetting<boolean>("image"),
+		]);
+
+	if (pathname === "/home") presenceData.details = "Browsing";
```

Comment:
I advise you to change to a switch for a better readability since you're using startsWith func every time.